spec/controllers/public/layers_controller_spec.rb

```diff
@@ -160,8 +199,8 @@ def trigger_show_geomap
 
         # Create 3 additional places with relations between them
         3.times do
-          place1 = FactoryBot.create(:place, :with_audio, layer_id: @layer.id, published: true)
-          place2 = FactoryBot.create(:place, layer_id: @layer.id, published: true)
+          place1 = FactoryBot.create(:place, :with_audio, layer_id: @layer.id, tag_list: %w[aaa bbb ccc], published: true)
+          place2 = FactoryBot.create(:place, layer_id: @layer.id, tag_list: %w[bbb ccc ddd], published: true)
```

Comment:
Ich bin nicht sicher was für SQL `acts_as_taggable` erzeugt und würde daher vorschlagen eine Query mit Tag-Filterung ebenfalls in dieses Sicherheitsnetz aufzunehmen.